### PR TITLE
_PyOpcache does not exist on Python 3.11

### PR DESCRIPTION
**Pull request by alex**

### CODE REVIEW

**Review by davidhewitt — APPROVED** [review]
Thanks!

File: pyo3-ffi/src/cpython/code.rs [hunk 1]
```diff
@@ -39,7 +39,7 @@ pub struct PyCodeObject {
     pub co_extra: *mut c_void,
     #[cfg(Py_3_8)]
     pub co_opcache_map: *mut c_uchar,
-    #[cfg(Py_3_8)]
+    #[cfg(all(Py_3_8, not(Py_3_11)))]
```
Comment (mejrs) [inline pyo3-ffi/src/cpython/code.rs:42]: With https://github.com/PyO3/pyo3/pull/2409 I split this struct's definition, so this cfg now does nothing.

Comment (davidhewitt) [reply pyo3-ffi/src/cpython/code.rs:42]: 👍 I'll tidy this up later.

Comment (davidhewitt) [reply pyo3-ffi/src/cpython/code.rs:42]: Done in #2432

**Pull request merged**